programs/survival/fakeGui.sc

```diff
@@ -0,0 +1,461 @@
+);
+
+__page(type_, creativeplayer, fakeplayer) -> (
+    if (
+        // 菜單
+        type_ == 'menu', (
+            screen = create_screen(creativeplayer, 'generic_9x3', fakeplayer~'display_name'+i18n(creativeplayer, 'who_menu'), _(screen, player, action, data, outer(fakeplayer)) -> (
+                slot = data:'slot';
+                if(
+                    action == 'close', (
+                        global_fakeplayersscreen:fakeplayer = null;
+                        drop_item(screen, -1);
+                    ),
+                    action == 'pickup', 
+                    if (
+                        slot > 9*3-1, null,
+                        (
+                            newPage = command = null;
+
+                            if (
+                                slot == 16, command = 'kill',
+                                slot == 15, command = 'stop',
+                                slot == 14, command = 'jump',
+                                slot == 13, command = 'swapHands'
+                            );
+
+                            if (
+                                command != null, (
+                                    global_fakeplayersscreen:fakeplayer = null;
+                                    run('player ' + (fakeplayer~'command_name') + ' ' + command);
+                                    soundDell(player);
+                                    close_screen(screen);
+                                ),
+                                slot == 12, newPage = 'bag',
+                                slot == 11, newPage = 'use',
+                                slot == 10, newPage = 'attack',
+                            );
+
+                            if (newPage != null, (
+                                soundDell(player);
+                                close_screen(screen);
+                                __page(newPage, player, fakeplayer);
+                            ));
+                            return('cancel'),
+                        ),
+                    ),
+                );
+            ));
+            global_fakeplayersscreen:fakeplayer = [screen, models];
+
+            for (l(
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'left_click')+'\', item: \'minecraft:wooden_sword\'}'),
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'right_click')+'\', item: \'minecraft:cooked_porkchop\'}'),
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'backpack')+'\', item: \'minecraft:chest\'}'),
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'switch_off-hand_items')+'\', item: \'minecraft:magenta_glazed_terracotta\'}'),
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'jump')+'\', item: \'minecraft:rabbit_foot\'}'),
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'stop')+'\', item: \'minecraft:barrier\'}'),
+                nbt('{ title: \'§f§L'+i18n(creativeplayer, 'remove')+'\', item: \'minecraft:tnt\'}'),
```

Comment:
You could create a `_item_icon` function for this, I think that would be cleaner.
```
_item_icon(creativeplayer, 'right_click', 'cooked_porkchop')
_item_icon(player, i18n_key, item_name)
```